.github/workflows/poetry-lock-command.yml

```
@@ -0,0 +1,144 @@
+        cache: 'poetry'
+
+    # Run `poetry lock`
+
+    - name: Run `poetry lock`
+      run: poetry lock
+
+    # Check for changes in git
+
+    - name: Check for changes
+      id: git-diff
+      run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)